modules/permutiveRtdProvider.js

```diff
@@ -125,6 +127,43 @@ function getDefaultBidderFn (bidder) {
       }
 
       return bid
+    },
+    pubmatic: function(bid, data, acEnabled) {
+      let segments = [];
+      var existingData = config.getBidderConfig(); // read existing bidder level configs if any
+      var dataUpdated = false;
+      if (Object.keys(existingData).length > 0) {
+        existingData = existingData['pubmatic'];
+        dataUpdated = existingData.isDataUpdated || false;
+      }
+
+      if (!dataUpdated) {
+        existingData = getNestedData(existingData, 'ortb2', 'user', 'data', 'segment');
+        segments = existingData ? segments.concat(existingData) : [];
+        // add AC specific segments
+        if (acEnabled && data.ac && data.ac.length > 0) {
+          for (var index in data.ac) {
+            segments = segments.concat({id: data.ac[index]});
+          }
+        }
+
+        if (!dataUpdated && segments && segments.length > 0) {
+          window['pbjs'].setBidderConfig({
```

Comment:
namespace can be changed by publisher so we can use global parameter for namespace

Reply:
Do we need to set the bidderConfig, can't we add it in Bid Object just like other partners?

Reply:
namespace change done.